.github/workflows/choco-release.yml

```diff
@@ -24,12 +24,17 @@ jobs:
       - name: Build the chocolatey package
         shell: powershell
         run: make choco
+      - name: Set CRC_VERSION env variable
+        shell: bash
+        run: |
+          crc_version=${{ github.event.release.tag_name }}
+          echo "CRC_VERSION=${crc_version:1}" >> $GITHUB_ENV
```

Comment:
Is this indirection required? It seems this:
```
        run: choco push ./packaging/chocolatey/crc/crc.${{ github.event.release.tag_name }}.nupkg --source https://push.chocolatey.org/`
```
or 
```
        run: choco push ./packaging/chocolatey/crc/crc.$GITHUB_REF_NAME.nupkg --source https://push.chocolatey.org/
```
both work

Or you could also use 
```
    env:
        CRC_VERSION: ${{ github.event.release.tag_name }}
```

Not clear why this has to go through GITHUB_ENV ?

Reply:
I tried the suggestions above https://github.com/crc-org/crc/compare/main...cfergeau:crc:ghactions in https://github.com/cfergeau/crc/actions/runs/5046786447/jobs/9052777467

Reply:
This could be done directly in the ` name: Push the choco to community.chocolatey.org` step though, and avoid going through GITHUB_ENV?

Reply:
yeah, it should be possible we have to use a powershell specific way of trimming the char, the shell for that step is `pwsh`

Reply:
For pushing, I don't think you need powershell? or the `choco` command has to be run from powershell? If you can't use bash to push, don't bother making any changes, the current version will do.

Reply:
@cfergeau PTALA :)